docs/install-index.md

```diff
@@ -3,6 +3,8 @@
 Install Percona XtraDB Cluster on all hosts that you are planning to use as cluster nodes
 and ensure that you have root access to the MySQL server on each one.
 
+We gather [Telemetry data] in the Percona packages and Docker images.
```

Comment:
Hi,
Could you please also add it to the pages:
* [Debian or Ubuntu](apt.md#apt) 
* Red Hat or CentOS](yum.md#yum)
* Running Percona XtraDB Cluster in a Docker Container